src/blocks/custom/form/components/form-email-options.js

```diff
@@ -0,0 +1,67 @@
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { TextControl, BaseControl } from '@wordpress/components';
+import { RichText } from '@wordpress/block-editor';
+
+export const FormEmailOptions = (props) => {
+  const {
+    richTextClass,
+    to,
+    subject,
+    message,
+    additionalHeaders,
+    onChangeTo,
+    onChangeSubject,
+    onChangeMessage,
+    onChangeAdditionalHeaders,
+  } = props;
+
+  const defaultMessage = message ?? 'Message from user [[name]]:<br><br>[[message]]';
+
+  return (
+    <Fragment>
+      {onChangeTo &&
+        <TextControl
+          label={__('To', 'eightshift-forms')}
+          help={__('Email address to which you wish to receive emails.', 'eightshift-forms')}
```

Comment:
Shouldn't this say 'send emails'?

Reply:
Hmm good point, it's more clear, will change. Although both are technically true (because the person setting up the forms is likely the person receiving them). 😄